packages/node/src/events/propose-install/operation.ts

```diff
@@ -7,15 +7,17 @@ import { getChannelFromPeerAddress } from "../../utils";
 export async function setAppInstanceIDForProposeInstall(
   selfAddress: Address,
   store: Store,
-  params: Node.InterNodeProposeInstallParams
+  params: Node.ProposeInstallParams,
+  appInstanceId: string,
+  respondingAddress: Address
```

Comment:
I think we should be consistent with using `string` and not `Address`

Reply:
This code base is consistently using `Address` (even though it's just a string type alias)

Reply:
OK -- maybe we can discuss later, not in scope for this pr.